src/pages/documentation/getting-started/validate.md

```diff
@@ -0,0 +1,41 @@
+# Validation and next steps
+
+As you begin to add solution APIs to your mobile implementation, you are going to want to validate that specific actions and experiences work as intended. Adding [Project Griffon](../beta/project-griffon/#what-can-project-griffon-do-for-you) to your application at the beginning of your implementation provides a way to quickly check to make sure the SDK has been instrumented properly and that data is flowing to Adobe Analytics and other solutions. FIX LINK
```

Comment:
There are multiple occurrences in this file that need to be renamed from Project Griffon or Griffon -> Assurance. Looks like they are already updated in gitbook

Reply:
Yeah, I was planning to change these when I do my second go through (including link updates, etc.)

Reply:
Ok, np. I think the rename already happened in gitbook, so this file was outdated compared with that too.